supertokens_python/recipe/emailpassword/asyncio/__init__.py

```diff
@@ -23,11 +23,12 @@ async def update_email_or_password(
     email: Union[str, None] = None,
     password: Union[str, None] = None,
     user_context: Union[None, Dict[str, Any]] = None,
+    apply_password_policy: Union[bool, None] = None,
```

Comment:
user_context should always be the last arg in the function - see the pattern everywhere